packages/commonwealth/server/models/notification.ts

```diff
@@ -61,6 +63,10 @@ export default (
       foreignKey: 'chain_id',
       targetKey: 'id',
     });
+    models.Notification.belongsTo(models.Thread, {
+      foreignKey: 'thread_id',
```

Comment:
if we remove foreign key related migration - no confusion with ordering of migration as well
just add new pre-computed column outside jsonb.
Thanks!

Reply:
@timolegros  @jnaviask  actually this will not impact anything, probably good to add.
We don't have this relationship in migration just at app level which is ok

this PR don't need any changes, just rebase against master and refresh migration timestamp, thanks!

Doing that now, thanks!

Reply:
This relationship **is** in the migration. Removing now.

Reply:
thanks i missed that one! not sure how!